README updated

j-tirelli opened the pull request.

j-tirelli committed:
README updated

j-tirelli committed:
Removed legacy code

j-tirelli committed:
Seperated seeder fn into fn with options (so far, write to file or write to mongo)

j-tirelli committed:
Initial data generation script.

j-tirelli committed:
Changed file to create a csv, ignored csv directly instead of folder so folder will be created.

luke-eastman commented on database/seed.js:
```diff
@@ -18,33 +19,65 @@ const generateRecords = function(numRecords) {
+  let numOfRecords = args[0] || 700;
+  let chunkSize = args[1] || numOfRecords;
+  let callback = callBackGenerator(args[2]);
+  let chunks = numOfRecords / chunkSize;
```
could chunks end up being a fraction here? maybe that wouldn't matter anyway.

j-tirelli replied:
Good Catch! I actually forgot to remove this file from my commit as it is going through a heavy revision now as well...

luke-eastman commented on database/seed.js:
```diff
@@ -18,33 +19,65 @@ const generateRecords = function(numRecords) {
+          console.log('error:', err);
+        } else {
+          console.log('example data file has been written');
+          debugger;
```
probably don't want the debugger in here

luke-eastman reviewed (approved):
Looks good! There are a few console logs sprinkled around but some of those are nice to have in a data generation script.

j-tirelli committed:
/database - Removed MongoDB from legacy code
package.json - Added dotenv and pg packages to
Added Postgres connections and helper files
/seed/dataGenerator - Moved heading line writer out of loop, changed default number of entries from 700 to 10,000,000
/server/index.js - Changed from callbacks to promises
